Obsidian.API/_Enums/CommandsIssuer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Obsidian.API
+{
+    [Flags]
+    public enum CommandIssuers
```

Comment:
File name does not match class name

Reply:
> Almost there. Also, did you check up with giant list of overrides of the ConsoleClient?

Oh, I forgot to delete ConsoleClient, it's no longer needed